### ISSUE TITLE
Cannot resolve method 'create(java.lang.String, char[], java.lang.String, java.lang.String)'

### ISSUE THREAD

**Issue by nihaozaoanwanan** [influxdata/influxdb-client-java#100]
InfluxDBClient influxDBClient = InfluxDBClientFactory.create("http://localhost:9999", token, org, bucket);
when i used this example，my idea telled me the following sentence.
Cannot resolve method 'create(java.lang.String, char[], java.lang.String, java.lang.String)'.
The example is used to connect to InfluxDB 2.0 .
how to run this example?

**Comment by bednar** [comment]
Hi @nihaozaoanwanan,

The method `create(java.lang.String, char[], java.lang.String, java.lang.String)` was introduced by #97. This will be released in [1.7.0](https://github.com/influxdata/influxdb-client-java/milestone/10) milestone at April 17, 2020.

Meanwhile you could use something like this:

```java
InfluxDBClientOptions options = InfluxDBClientOptions.builder()
        .url("http://localhost:9999")
        .authenticateToken(token)
        .org(org)
        .bucket(bucket)
        .build();

InfluxDBClient influxDBClient = InfluxDBClientFactory.create(options);
```

Regards

**Comment by nihaozaoanwanan** [comment]
thanks
